docs/DOCKER.md

```diff
@@ -0,0 +1,39 @@
+Oengus Docker instructions
+==========================
+
+To run Oengus backend for development purposes, you don't need to use Docker. Using IntelliJ IDEA for development is
+both easier and more convenient: [IntelliJ IDEA instructions](./DEVELOPER.md). However, if you do need to use the Docker
+image, follow the instruction below on how to use Oengus in Docker in Ubuntu or other Debian-based GNU/Linux
+distributions.
+
+Installation
+------------
+
+Install packages listed in "Prerequisites" section of [DEVELOPER.md](./DEVELOPER.md). As of March 2022, Ubuntu's
+docker-compose is way outdated. Because of that, you'll have to install Docker from Docker's packages:
+https://docs.docker.com/engine/install/ubuntu/
+
+Docker setup
+------------
+
+If you haven't used Docker before, you need to do the following:
+
+1. Add yourself to the `docker` group to have access to the daemon:
+
+       $ sudo usermod -a -G docker $USER
+2. To get the updated groups in text shells you can just re-log in, but in GUI shells, you'll have to re-log in via
+   their GUI.
+
+       $ sudo su - $USER
+3. Start up the docker daemon (assuming systemd-based distro):
+
+       $ sudo systemctl start docker
+
+Get Oengus Docker image
+-----------------------
+
+Get the Docker image of Oengus from [Docker Hub][Docker image]:
+
+    $ docker pull oengusio/backend
```

Comment:
Something to keep in mind is that self-hosted instances are not allowed to use the Oengus branding (nor will they receive support).
This probably won't be an issue with the backend but it feels good to mention both these things.

Also, codeblocks :)

Reply:
Because I'm not familiar with the branding issue, I don't understand what you are suggesting here. Do you want me to put some kind of disclaimer in `docs/DOCKER.md`?

Reply:
Yup. There needs to be a disclaimer that they are not allowed to use the branding

Reply:
I would personally say that at the top of "How to run on your computer" would be a better place

Reply:
Added a disclaimer in 9069865973ebe5f7e254d615a4f88dc79f39b4e2.